package.json

```diff
@@ -24,7 +24,12 @@
     "onCommand:browser-preview.openPreview",
     "onDebugInitialConfigurations",
     "onDebug",
-    "onFileSystem:vsls"
+    "onFileSystem:vsls",
+    "onCommand:liveshare.start",
```

Comment:
This and the following lines ensure that the Browser Preview extension is activated as soon as the host starts a Live Share session.

Reply:
We also have the `startFromFileTreeExplorer`/`startFromActivityBar` commands.

Reply:
Ohh ignore my comment.